src/steps/run.go

```diff
@@ -44,6 +44,7 @@ func Run(runState *RunState, repo *git.ProdRepo, driver drivers.CodeHostingDrive
 		if runErr != nil {
 			runState.AbortStepList.Append(step.CreateAbortStep())
 			if step.ShouldAutomaticallyAbortOnError() {
+				cli.PrintError(fmt.Errorf(runErr.Error() + "\n" + "Auto-aborting..."))
```

Comment:
You could merge the two strings:

```suggestion
				cli.PrintError(fmt.Errorf(runErr.Error() + "\nAuto-aborting..."))
```